newIDE/app/src/SceneEditor/index.js

```diff
@@ -850,8 +863,127 @@ export default class SceneEditor extends React.Component<Props, State> {
     if (this.editor) this.editor.zoomBy(-0.1);
   };
 
-  _onContextMenu = (x: number, y: number) => {
-    if (this.contextMenu) this.contextMenu.open(x, y);
+  _onContextMenu = (
+    x: number,
+    y: number,
+    ignoreSelectedObjectNamesForContextMenu?: boolean = false
+  ) => {
+    if (!ignoreSelectedObjectNamesForContextMenu) {
+      if (this.contextMenu) this.contextMenu.open(x, y);
+    } else {
+      this.setState({ ignoreSelectedObjectNamesForContextMenu: true }, () => {
```

Comment:
a bit "dirty", I wonder if we should allow some arbitrary parameters to be passed as third option to `contextMenu.open` that would be then passed to buildMenuTemplate? :)

Reply:
Yes it is dirty.

I thought about that as well, but I didn't want to change the signature for this often-used method. But this is a case for which it's worth I think.